app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :likes, foreign_key: :user_id
+  has_many :comments, foreign_key: :user_id
```

Comment:
- [ ] @simplegoose Great work adding association in your user model, however, you forgot to add a custom method that returns the 3 most recent posts for a given user.